### PR TITLE
Phillip/fix custom action hub

### PR DESCRIPTION
**Pull request by phillipperalez**
This change will fix to the action hub repo "looker/actions" to correctly build with a "yarn install"

### CODE REVIEW

File: src/hub/action_request.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import * as express from "express"
 import * as oboe from "oboe"
 import * as httpRequest from "request"
-import * as sanitizeFilename from "sanitize-filename"
 import * as semver from "semver"
 import { PassThrough, Readable } from "stream"
 import * as winston from "winston"
 import { formatToFileExtension, truncateString } from "./utils"
 
+let sanitizeFilename = require("sanitize-filename")
```
Comment (brln-looker) [inline src/hub/action_request.ts:9]: can you do this as an import for consistency?

Comment (phillipperalez) [reply src/hub/action_request.ts:9]: When this particular library updated it just stopped working in the `import` style unfortunately. This was a known issue that they were fine with. It was causing ECMAinterop issues when used in the different styles of `import` I could think of

Comment (brln-looker) [reply src/hub/action_request.ts]: you tried `import * as sanitize-filename`?

Comment (phillipperalez) [reply src/hub/action_request.ts]: yes and that shows up with the interop error

**Pull request merged**